### PR TITLE
asf-tools v0.8.1

### PR DESCRIPTION
**Pull request by jtherrmann**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (jtherrmann) [inline recipe/meta.yaml:10]:
```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

**Comment by jtherrmann** [comment]
@conda-forge-admin, please rerender

**Review by AndrewPlayer3 — APPROVED** [review]
Version number and hash matches [PyPI](https://pypi.org/project/asf-tools/#asf_tools-0.8.1.tar.gz)

**Comment by jtherrmann** [comment]
@conda-forge-admin, please restart ci

**Pull request merged**